src/components/confirm-input.tsx

```diff
@@ -10,7 +10,8 @@ export default function ConfirmInput({
   loading = false,
   disabled = false,
   autoFocus = true,
-  onConfirm = () => null,
+  onTextChange = text => null,
+  onConfirm = command => null,
```

Comment:
No unused vars. Please run 'pnpm lint' before submitting the code. Later, I will add husky to this project.